#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_check_python_2.5.dpatch by Anthony Fok <foka@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch to check for Python 2.5 during configure.
## DP: after applying "make -f admin/Makefile.common" have to be performed

@DPATCH@

--- qterm-0.4.1.orig/admin/acinclude.m4.in
+++ qterm-0.4.1/admin/acinclude.m4.in
@@ -4361,16 +4361,18 @@
 
 AC_DEFUN([KDE_CHECK_PYTHON],
 [
-  KDE_CHECK_PYTHON_INTERN("2.4",
-   [KDE_CHECK_PYTHON_INTERN("2.3", 
-     [KDE_CHECK_PYTHON_INTERN("2.2", 
-       [KDE_CHECK_PYTHON_INTERN("2.1", 
-         [KDE_CHECK_PYTHON_INTERN("2.0", 
-           [KDE_CHECK_PYTHON_INTERN($1, $2) ])
+  KDE_CHECK_PYTHON_INTERN("2.5",
+   [KDE_CHECK_PYTHON_INTERN("2.4",
+     [KDE_CHECK_PYTHON_INTERN("2.3", 
+       [KDE_CHECK_PYTHON_INTERN("2.2", 
+         [KDE_CHECK_PYTHON_INTERN("2.1", 
+           [KDE_CHECK_PYTHON_INTERN("2.0", 
+             [KDE_CHECK_PYTHON_INTERN($1, $2) ])
+	   ])
          ])
        ])
      ])
-  ])
+   ])
 ])
 
 AC_DEFUN([KDE_CHECK_STL],
